terraform/modules/database/main.tf

```diff
@@ -0,0 +1,23 @@
+resource "aws_db_instance" "api3-operations" {
+  identifier             = "api3-operations"
+  instance_class         = "db.t3.micro"
```

Comment:
Recently RDS added [support for t4g instances](https://aws.amazon.com/about-aws/whats-new/2021/09/amazon-rds-t4g-mysql-mariadb-postgresql/). They basically offer up to 36% better price performance over t3 variants.